crabs/detection_tracking/train_model.py

```diff
@@ -61,17 +60,11 @@ def load_config_yaml(self):
         with open(self.config_file, "r") as f:
             self.config = yaml.safe_load(f)
 
-    def set_run_name(self):
-        self.run_name = set_mlflow_run_name()
-
-    def setup_logger(self) -> MLFlowLogger:
+    def setup_trainer(self):
         """
-        Setup MLflow logger for training, with checkpointing.
-
-        Includes logging metadata about the job (CLI arguments and SLURM job IDs).
+        Setup trainer with logging and checkpointing.
         """
-        # Assign run name
-        self.set_run_name()
+        self.run_name = set_mlflow_run_name()
```

Comment:
👍,  we need to remember to change the evaluation one too to be consistent (probs best in the evaluation PR)